com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -1389,41 +1387,32 @@ private bool ExportInstance(GameObject unityGo, FbxScene fbxScene, FbxNode fbxNo
                 return false;
             }
 
-            Object unityPrefabParent = PrefabUtility.GetCorrespondingObjectFromSource(unityGo);
```

Comment:
interesting so the prefab parts were not necessary in the end?

Reply:
The prefab id was just used to grab an fbx mesh, which can also be done with a unity mesh. The id was never assigned to the mesh itself.